src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // const MEN = people.filter(person => century
+  //   ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+  //   : person.sex === 'm'
+  // );
+
+  const MEN = century
+    ? people.filter(person => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century)
```

Comment:
```suggestion
    ? people.filter(person => person.sex === SEX_MALE
      && Math.ceil(person.died / YEARS_IN_CENTURY) === century)
```
Consider replacing all magic values with constants. Declare them at the top of the file.